python/tk_hiero_export/sg_audio_export.py

```diff
@@ -184,8 +186,33 @@ def _baseTaskStep(self, item):
                         (item.timelineOut() + handles) + 1,
                     )
 
-                    # If trackitem write out just the audio within the cut
-                    self._sequence.writeAudioToFile(self._audioFile, start, end)
+                    # Versions of Nuke >= 13  require the additional parameters 3 and 2 to be passed to
+                    # the writeAudioToFile method.
+                    # Check Nuke´s version
+                    nuke_version = (
+                        nuke.NUKE_VERSION_MAJOR,
+                        nuke.NUKE_VERSION_MINOR,
+                        nuke.NUKE_VERSION_RELEASE,
+                    )
+
+                    if nuke_version[0] >= 13:
+                        # Retrieving the bithDepth and bitRate values for nuke version >= 13
+                        bitDepth_data = self._initDict["preset"]._properties["bitDepth"]
```

Comment:
Retrieving these values only when the version of Nuke is >=13 now, tested on nuke 13.1v2, now testing on nuke 13.0v6